app/controllers/AnalysisController.scala

```diff
@@ -0,0 +1,143 @@
+package controllers
```

Comment:
Refactor this controller for getting data for Knolx analysis page from different rest end points.